bootstrap/Makefile

```diff
@@ -19,7 +19,7 @@ GOPATH ?= $(HOME)/go
 # export DOCKER_BUILD_OPTS=--no-cache
 BOOTSTRAPPER_IMG ?= gcr.io/$(GCLOUD_PROJECT)/bootstrapper
 KFCTL_IMG ?= gcr.io/$(GCLOUD_PROJECT)/kfctl
-TAG ?= $(eval TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)-$(shell git diff | shasum -a256 | cut -c -6))$(TAG)
+TAG ?= $(eval TAG := $(shell git describe))$(TAG)
```

Comment:
I think we want

```
git describe --tags --long
```
--tags is needed because I don't think GitHub is creating annotated tags.

--long is needed because I think we should print out the actual commit. 

On master

```
git describe --tags --long
v0-29-g9625cde8
```

Not sure where the v0 tag is coming from

On v0.5-branch

```
git describe --tags --long
v0.5.0-0-ge7d0b5a7
```

Reply:
Looks like someone created the v0 tag.
https://github.com/kubeflow/kubeflow/tree/v0